ern-local-cli/src/commands/platform/config.js

```diff
@@ -28,6 +28,7 @@ exports.handler = function ({
         : value
 
       ernConfig.setValue(key, valueToset)
+      log.info(`Done! ${key}: ${ernConfig.getValue(key)}`)
```

Comment:
${key} set to ${ernConfig.getValue(key) , that way we convey the message of the method purpose. minor suggestion.

Reply:
Done.